apps/files/src/components/FileSharingSidebar.vue

```diff
@@ -258,7 +258,7 @@ export default {
     },
     noResharePermsMessage () {
       const translated = this.$gettext('You don\'t have permission to share this %{type}.')
-      return this.$gettextInterpolate(translated, { type: this.highlightedFile.type })
+      return this.$gettextInterpolate(translated, { type: this.highlightedFile.type }, false)
```

Comment:
does it hurt keeping "true" here ? just wondering

Reply:
The only values we are getting here are "file" and "folder" so don't see much value in keeping it there.